src/Package.zig

```diff
@@ -617,7 +690,7 @@ const ReadableResource = struct {
         pkg_prog_node: *std.Progress.Node,
     ) !PackageLocation {
         switch (rr.resource) {
-            inline .file, .http_request => |*r| {
+            inline .file, .http_request, .git_fetch_stream => |*r| {
```

Comment:
I was a bit concerned with how much code is inside this inline switch case, causing bloat, and that sent me on a wild yak chase, ultimately resulting in https://github.com/ziglang/zig/pull/17344 which got blocked by  #17343.

Conclusion? idk. This can be merged, but I want to spend some time de-bloatifying things later.